test/sanity/validate-modules/ignore.txt

```diff
@@ -161,6 +161,7 @@ lib/ansible/modules/cloud/google/gcpubsub_facts.py E322
 lib/ansible/modules/cloud/google/gcpubsub_facts.py E324
 lib/ansible/modules/cloud/google/gcpubsub_facts.py E326
 lib/ansible/modules/cloud/google/gcspanner.py E322
+lib/ansible/modules/cloud/kubevirt/kubevirt_vm.py E325
```

Comment:
Why does this PR add lines to ignore.txt? Isn't the whole point to get rid of these?

Reply:
Caused by a rebase.

Reply:
The original PR (and the ones we break out because of an issue) were all related to doc_fragments only. Since an unrelated update impacted the change, I simply documented the new inconsistency. The issues in the modules are for a later date (I am doing some of these in other PRs) for everything related to Working Groups I manage.